Geometry_Engine/Create/ShapeProfile.cs

```diff
@@ -194,5 +481,19 @@ private static List<ICurve> MirrorAboutLocalZ(this List<ICurve> curves)
         }
 
         /***************************************************/
+
+        private static void InvalidRatioError(string inputs)
+        {
+            Engine.Reflection.Compute.RecordError("The ratio between " + inputs + " makes section inconceivable");
+        }
+
+        /***************************************************/
+
+        private static void InvalidRatioError2(string first, string second)
+        {
+            Engine.Reflection.Compute.RecordError("The ratio between " + first + " in relation to " + second + " makes section inconceivable");
```

Comment:
Know I kind of wrote this, but would change this to

"The ratio of the " + first + " in relation to the " + second + " makes section inconceivable"